api/api.go

```diff
@@ -149,7 +149,7 @@ func (b *BlockChainAPI) Syncing(ctx context.Context) (interface{}, error) {
 		return handleError[any](err, b.logger, b.collector)
 	}
 
-	if currentBlock == highestBlock {
+	if currentBlock >= highestBlock {
```

Comment:
Nice catch :100: 
Currently, `GetLatestEVMHeight`, will execute a Cadence script to fetch the latest EVM height, against the latest sealed execution state. Since there's no way to execute it against unsealed Flow blocks, this check should be fine :+1: 
Maybe it's worth adding a comment though.